worlds/dark_souls_3/Locations.py

```diff
@@ -688,6 +688,273 @@ def get_name_to_id() -> dict:
+        "FS: Uchigatana", # NPC drop
+        "FK: Soul of a Stray Demon",
+        "US: Irithyll Straight Sword", # Irithyll Outrider drop
+        "CC: Soul of a Demon",
+        "SL: Lightning Stake", # Sand Worm drop
+        "SL: Fume Ultra Greatsword", # Knight Slayer Tsorig drop
+        "CD: Aldrich's Sapphire", # Deep Accursed drop
+        "CD: Spider Shield", # NPC drop
+        "CD: Saint Bident", # Guarded by giant
+        "IBV: Pontiff's Right Eye", # Sulyvahn's Beast drop
+        "IBV: Ring of Favor", # Sulyvahn's Beast Duo drop
+        "IBV: Aldrich Faithful", # Sulyvahn's Beast Duo reward
+        "AL: Aldrich's Ruby", # Deep Accursed drop
+        "PC: Logan's Scroll", # NPC drop
+        "LC: Irithyll Rapier", # Boreal Outrider drop
+        "GA: Crystal Scroll", # Crystal Sage drop
+        "GA: Outrider Knight Helm",
+        "GA: Outrider Knight Armor",
+        "GA: Outrider Knight Gauntlets",
+        "GA: Outrider Knight Leggings",
+        "GA: Golden Wing Crest Shield", # NPC drop
+        "GA: Onikiri and Ubadachi", # NPC drop
+        "GA: Sage's Crystal Staff", # NPC drop
+        "AP: Dragon Tooth", # NPC drop
+        "DH: Flame Fan", # Desert Pyromancer Zoey drop
+        "RC: Iron Dragonslayer Helm",
+        "RC: Iron Dragonslayer Armor",
+        "RC: Iron Dragonslayer Gauntlets",
+        "RC: Iron Dragonslayer Leggings",
+        "RC: Crucifix of the Mad King", # Shira drop
+    ]),
+
+    "Key Locations": frozenset([
+        "HWL: Cell Key",
+        "ID: Jailer's Key King",
+        "ID: Jailbreaker's Key",
+        "ID: Old Cell Key",
+        "LC: Grand Archives Key",
+        "PW: Contraption Key",
+
+        # Not currently randomized:
+        # "FS: Grave Key",
+        # "FS: Lift Chamber Key",
+        # "FS: Tower Key",
+    ]),
+
+    "Guarded by Keys": frozenset([
+        # Guarded by Tower Key
+        "FSBT: Covetous Silver Serpent Ring",
+        "FSBT: Fire Keeper Robe",
+        "FSBT: Fire Keeper Gloves",
+        "FSBT: Fire Keeper Skirt",
+        "FSBT: Estus Ring",
+        "FSBT: Fire Keeper Soul",
+
+        # Guarded by Cell Key
+        "HWL: Greirat's Ashes",
+
+        # Guarded by Jailer's Key Ring
+        "ID: Prisoner Chief's Ashes",
+        "ID: Karla's Ashes",
+        "ID: Karla's Pointed Hat",
+        "ID: Karla's Coat",
+        "ID: Karla's Gloves",
+        "ID: Karla's Trousers",
+
+        # Guarded by Jailbreaker's Key
+        "ID: Bellowing Dragoncrest Ring",
+
+        # Guarded by Old Cell Key
+        "ID: Covetous Gold Serpent Ring",
+    ]),
+
+    # Locations that are particularly tricky to find or get to, for players
+    # without an encyclopedic knowledge of DS3 who don't want to get stuck
+    # looking for an invisible wall or one random mob with a guaranteed drop.
+    "Hidden Locations": frozenset([
+        # TODO: this list isn't at all comprehensive, expand it
+
+        "DC: Dragonhead Shield", # requires gesture
```

Comment:
```suggestion
        "RC: Dragonhead Shield", # requires gesture
```
Fixed to match the actual location name